ndbench-cockroachdb-plugins/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+
+dependencies {
+    compile project(':ndbench-api')
+    compile project(':ndbench-core')
+
+    compile 'javax.inject:javax.inject'
+    testCompile group: 'com.netflix.archaius', name: 'archaius2-guice', version: '2.1.11'
+    // https://mvnrepository.com/artifact/org.postgresql/postgresql
```

Comment:
I would add 1 line comment here on why we are adding postgresql dep, `Cockroach DB uses PostgresSQL wire protocol - Reference link to documentation `